site/content/en/references/kustomize/kustomization/images/_index.md

```diff
@@ -161,6 +161,26 @@ kustomize edit set image foo:$FOO_IMAGE_TAG
 kubectl apply -f .
 ```
 
+## Use the full image name to modify the name, tag and digest
+
+Kustomize has the ability to set numerous fields on images in a single, idempotent command.
+
+```bash
+kustomize edit set image index.docker.io/oldregistry/myapp=quay.io/newregistry/myapp:v2@sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3
+```
+
+Output:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+  - digest: sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3
+    name: index.docker.io/sourcegraph/redis-store:insiders
+    newName: quay.io/my-reg/myapp:v
```

Comment:
This and the above don't seem to match the arguments in the kustomize invocation. What version of Kustomize are you using to exercise this? I tried to validate it with a couple different versions, and kept running into https://github.com/kubernetes-sigs/kustomize/issues/4081